src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,5 +1,18 @@
 package mate.academy
 
+import mate.academy.model.Figure
+import mate.academy.service.FigureSupplier
+
+private const val ARRAY_SIZE = 6
+private const val HALF = 2
 fun main() {
-    // draw all figures here
+   val figureArr : Array<Figure?> = arrayOfNulls(ARRAY_SIZE)
+   for (i in 0 until figureArr.size.div(HALF)) {
```

Comment:
Could be simplified with forEachIndexed
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/for-each-indexed.html
and if()